test/AllPairs/test_ap.py

```diff
@@ -0,0 +1,84 @@
+import shutil
+from pathlib import Path
+
+import pytest
+
+from src.allpairs import AllPairs
+
+TEST_DIR = 'test/AllPairs/'
+OUT_FILE = TEST_DIR+'output/out.txt'
+
+EXPECTED_FILE = TEST_DIR+'/expected/out.txt' ## TODO not currently checked.
```

Comment:
Is this the expected output for `sample-in-net.txt` and `sample-in-nodetypes.txt`? We should give it a `sample-in` prefix to distinguish it from the `correctness` inputs.